### PR TITLE
Echoes: document GWC terminal fall to item

### PR DESCRIPTION
**Pull request by JeffGainsNGames**

### CODE REVIEW

**Comment by JeffGainsNGames** [comment]
bump.

File: randovania/games/prime2/logic_database/Temple Grounds.txt [hunk 1]
```diff
@@ -1441,11 +1441,9 @@ Extra - in_dark_aether: False
...
           Any of the following:
               Grapple Beam or Use Screw Attack (Space Jump)
-              All of the following:
-                  Space Jump Boots
-                  Any of the following:
-                      Scan Visor and Combat/Scan Dash (Advanced)
-                      Terminal Fall Abuse (Beginner) and Normal Damage ≥ 10
+              Scan Visor and Space Jump Boots and Combat/Scan Dash (Advanced)
+              # Jump to center platform, then towards platform with the cannon https://youtu.be/_Cs-JtpIuNE
+              Space Jump Boots and Terminal Fall Abuse (Beginner) and Normal Damage ≥ 10
```
Comment (XenoWars) [inline randovania/games/prime2/logic_database/Temple Grounds.txt:1446]: It is good, but an argument can be made that it should require Standable Terrain. Would like others' thoughts.

**Review by dyceron — APPROVED** [review]

File: randovania/games/prime2/logic_database/Temple Grounds.txt (hunk 1 above)
Comment (dyceron) [reply randovania/games/prime2/logic_database/Temple Grounds.txt:1446]: Standable Terrain for whatm

Comment (JeffGainsNGames) [reply randovania/games/prime2/logic_database/Temple Grounds.txt:1446]: standing on the center platform https://discord.com/channels/458432177223368704/499458280108589061/1260277202818302063
my pov was that it's an intentional platform and doesn't need the trick but I won't fight back if folks disagree

**Pull request merged**